### ISSUE TITLE
Change <a> tag on Reset Score to Button

### ISSUE THREAD

**Issue by rrcampbell-exe** [rrcampbell-exe/real-fake-birds#32]
For accessibility reasons, the Reset Score link should be made into a button and then styles to look like its current state.

**Comment by rrcampbell-exe** [comment]
Closed by PR #36.
